internal/pkg/template/input/input.go

```diff
@@ -94,4 +94,23 @@ func (i Input) Available(params map[string]interface{}) bool {
 	return result.(bool)
 }
 
-type Option string
+type Option struct {
+	Id   string `json:"id" validate:"required"`
+	Name string `json:"name" validate:"required"`
+}
+
+func MapOptionsByName(options []Option) map[string]string {
```

Comment:
Bolo by lepsie definovat typ `type Options []Option`, a nad nim tieto metody, `ByName`, `Names`.
Zbytocne to nebude vyskakovat v IDE v situaciach, ked sa to neda pouzit.